GameOfLife/GameOfLife/Services/FileIO.cs

```diff
@@ -115,18 +128,37 @@ private GameFieldModel ConvertListOfRowsToGameField(List<string> inputList)
                 }
             }
 
+            _mainEngine.SavedGameLoaded = true;
             _stringList.Clear();
-            return gameField;
         }
 
         /// <summary>
-        /// Method to save the current games state to a text file.
+        /// Method to read the generation number from the file.
+        /// </summary>
+        /// <param name="inputList">List of the Game Field cell rows.</param>
+        /// <returns>Returns the generation number in the string form.</returns>
+        private static string TakeGenerationNumberFromFile(List<string> inputList)
```

Comment:
VS suggested me that I can make some methods 'static' because they are not using global variables or something like that. I then checked on stackoverflow that it is okay to do so. Is it a bad practice after all and should it be avoided?

Reply:
I dont think its bad practice per se, was just wondering, why the static was there. In practice, i would say, we try to avoid using statics, as using them can get out of hand real quick and in this case, you had a private method, that is being called in the same class, by another method and since that other method does not have a static keyword, you could call this method without the static keyword as well.

Reply:
Understood. I removed static keyword from methods.